app/helpers/candidates_helper.rb

```diff
@@ -17,102 +17,143 @@ def candidate_display_activity(event)
     actor_account_name = compose_actor_account_name(event)
 
     text =
-      if event.type == "email_received"
-        []
-      else
-        [actor_account_name]
-      end
-    text <<
       case event.type
       when "candidate_added"
-        "added the candidate"
+        if event.properties["method"].in?(%w[manual applied])
```

Comment:
Looks like we mixed the initial idea and result of discussion. On the call we agreed to not show "manually" when candidate applied, since "FreeATS added manually" doesn't make sence